src/explorer/utils/validations.ts

```diff
@@ -16,22 +16,23 @@ export function inputValidation(value: string, key: string): string {
     "free_sru",
     "free_hru",
   ];
-  const textualFields: string[] = [
-    "countryFullName",
-    "farmingPolicyName",
-    "certificationType",
-    "farmName",
-    "pricingPolicyId",
-    "country",
-  ];
-  const specialChars = /[`!@#$%^&*()+\-=[\]{};':"\\|,.<>/?~]/;
+  const textualFields: string[] = ["farmingPolicyName", "certificationType", "farmName", "pricingPolicyId"];
+
+  const countryFields: string[] = ["countryFullName", "country"];
+  const specialChars = /[ `!@#$%^&*()+\-=[\]{};':"\\|,.<>/?~]/;
+  const countryRegex = /^[a-zA-Z\s]*$/;
```

Comment:
hmm, the countryRegex can be empty

Reply:
added a condition to validate empty values

Reply:
I think it can be changed to `[a-zA-Z]+[a-zA-Z\s]*`

Reply:
Done.